### DISCUSSION TITLE
kpanda/user-guide/clusterops/latest-operations/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1290] [Announcements]
# kpanda/user-guide/clusterops/latest-operations/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/clusterops/latest-operations/

**Comment by liyuerich** [comment]
集群操作是哪些具体的操作？
